packages/util/src/api.ts

```diff
@@ -37,38 +38,55 @@ export class Api extends BaseApi {
   public readonly referralSystem: ReferralSystemModule = new ReferralSystemModule(this);
   public readonly assets: AssetsModule = new AssetsModule(this);
 
-  // # History methods
-
-  public get accountHistory(): Array<History> {
-    return this.history.filter(({ type }) => !isBridgeOperation(type));
-  }
-
   public initAccountStorage() {
     super.initAccountStorage();
     this.bridge.initAccountStorage();
 
-    // transfer old history to accountStorage
-    if (this.storage) {
-      const oldHistory = JSON.parse(this.storage.get('history')) || [];
+    // since 1.7.15 history should be saved as object
+    if (this.accountStorage) {
+      const oldHistory = this.history;
 
-      if (oldHistory.length) {
-        this.history = oldHistory;
+      if (Array.isArray(oldHistory)) {
+        this.runHistoryListToObjectMigration(oldHistory);
       }
+    }
+  }
 
-      this.storage.remove('history');
+  // # History methods
+  /**
+   * Save history items in storage as object
+   * @param list array of history items
+   */
+  private runHistoryListToObjectMigration(list: Array<HistoryItem>) {
+    const history = {};
+    const bridgeHistory = {};
+
+    for (const item of list) {
+      if (!item.id) continue;
+      if (isBridgeOperation(item.type)) {
```

Comment:
I think it's better to leave this name, because to determine the membership in any group of operations, we use the name 'is[group]Operation' like `isBridgeOperation`, `isLiquidityPoolOperation`